etc/battlePresets.conf

```diff
@@ -87,11 +87,12 @@ disabledunits:-*
 startpostype:1|0|2
 
 [tourney] 
-description:Tournament 1v1 Game Battle Settings
+description:Tournament Game Battle Settings
 resetoptions:1
 disabledunits:-*
 startpostype:2|0|1
 allowuserwidgets:0
+ranked_game:0
```

Comment:
Why not ranked?

Reply:
It makes no sense to have ranking enabled in tournament, just the same as autobalance: by nature the teams are prepared in advance and matches are "unfair" from the POV of the balancer / ranking system.

Keeping ranking enabled generates pressure on players:

- Weak players / teams might be deterred from joining tournaments if they care about their OS, as they know in advance they will lose to everybody else.
- Strong players with weak friends might be deterred from forming a team if they care about their OS, as they will lose a lot of OS if their team loses against a more uniformly but lower rated team (think 30+ OS with 10 OS friends vs. a team of 20 OS).

It also generates unrealistic OS data points, as it is unlikely any player / team would normally engage in a match they know in advance they will lose (i.e. nobody 10 OS would ever accept to play a ranked match where one team is all 10 OS and the other is all 30 OS), as it would basically be boosting.

Reply:
OS models doesn't care about this much, it can properly rate not balanced matches, the only thing that matters is that players actually try to win, and that precondition is met during tournament.

I was testing it myself some time ago https://colab.research.google.com/drive/1s3VupikIQNJ7Bqn6OzxziP0XeFQzRmJE

Reply:
This is correct from a rating perspective, but since OS is also used as the ranking metric displayed to players for matches in general, I still think it should be unranked. It's a bit like the "silos" situation with maps etc. where it was discussed already that for proper rating to occur we would almost need a rating per map. Tournaments are specific encounters involving prefixed teams, I think they are different enough from regular matches to warrant being separated (again just like most people will disable ranking if they try to get a 10 OS vs 30 OS game going).

As a side note, this is also the same in other eSports events, tournaments are never played with ranking enabled (e.g. LoL, Dota, CS:GO, ...). Typically what happens instead is there are dedicated tournament rankings established for recurring events (e.g. HLTV, ESL) and generally tied to teams rather than individuals, but we're not quite there yet for BAR 😛

Reply:
(As a second side note: for Nation Wars we had instances of players asking for ranking to be disabled because they did not want to play with it enabled... just to give some more support to the "psychological" aspect of it)

Reply:
Okey, maybe as a compromise let's just leave it unspecified to allow both? This way it can be configured per tournament?

If you feel strongly about this being off, I would love to get input from @PtaQQ here as I don't want to decide on it myself.

Reply:
I can see the argument for unranked being default for team tourneys. For example I play Nation Wars for fun, knowing that we probably are not going for a win (don't tell the boiz :p) and losing rating this way does hurt. I think it's a good call. 

However in 1v1 I don't see a reason for unranked. Can the preset adjust it dynamically depending on the team size? A lot of good data gets collected during tourneys and many players play 1v1 almost exclusively on tourneys so it's good these stay ranked imo. 

If the distinction is non trivial or would require separate presets I would go for unspecified and just let tourney organisers decide per event.

Reply:
Good point about 1v1 tournaments making up a large number of games (or even all of them) for some players because they don't play much 1v1 outside of tournaments. I don't think dynamic adjustment is possible sadly, unless there's some kind of SPADS mechanism I don't know about.

However maybe separate presets would still make sense? The 1v1 preset could keep the team size set to 1 with ranked enabled (so, exactly the current preset we have), while the team preset would lift off the team size and remove ranked (as proposed in this PR). I think it's OK, it's not like we have limits on the number of presets and it's not like we have "too many presets" already, do we?